WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -24,6 +24,11 @@ final class ManualTrackingViewController: UIViewController {
         return noticePresenter
     }()
 
+    private lazy var keyboardFrameObserver: KeyboardFrameObserver = {
```

Comment:
This is one of those times when I wish we could declare vars in extensions... 🤷‍♂